### PR TITLE
community: Add bio for Răzvan Deaconescu

### PR DESCRIPTION
**Pull request by razvand**
Add bio and picture for Răzvan Deaconescu

### CODE REVIEW

File: website/docs/community/people.md [hunk 1]
```diff
@@ -4,4 +4,16 @@ title: People
 
 # Project Team
 
-This is content
+## Răzvan Deaconescu
+
+![Răzvan Deaconescu](../../static/img/people/razvan_deaconescu.png)
+
+Răzvan is the OpenEdu project (21-COP-0016) director.
+He supervises project activities and is engaged in content development for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/) and the [Security Summer Schoo](https://github.com/security-summer-school/).
```
Comment (teodutu) [inline website/docs/community/people.md]:
```suggestion
He supervises project activities and is engaged in content development for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/) and the [Security Summer School](https://github.com/security-summer-school/).
```

File: website/docs/community/people.md [hunk 2]
```diff
@@ -4,4 +4,16 @@ title: People
...
+He's in charge with engaging the teaching staff at University POLITEHNICA of Bucharest and other universities and educational institutions in Romania.
+
+Răzvan is an Associate Professor at University POLITEHNICA of Bucharest, Romania, the Computer Science and Engineering Department.
```
Comment (teodutu) [inline website/docs/community/people.md]:
```suggestion
Răzvan is an Associate Professor at the University POLITEHNICA of Bucharest, Romania, the Computer Science and Engineering Department.
```

**Review by teodutu — CHANGES_REQUESTED** [review]
You made a few typos.

File: website/docs/community/people.md (hunk 2 above)
Comment (lizababu) [reply website/docs/community/people.md]: I'd say it's correct without `the`.

**Review by lizababu — APPROVED** [review]
LGTM

**Review by teodutu — APPROVED** [review]
👍

**Review by lizababu — APPROVED** [review]
Nice pic!

**Pull request merged**